google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryRequestInfo.java

```diff
@@ -49,9 +50,9 @@
     this.maxResults = config.getMaxResults();
     this.query = config.getQuery();
     this.queryParameters = config.toPb().getQuery().getQueryParameters();
+    this.requestId = UUID.randomUUID().toString();
```

Comment:
Are there any cases where a QueryRequestInfo may get reused? Such usage may indicate we set this at time of sending the request (outside of the retry wrapper) rather than time of instantiating the request.

Reply:
Yes -- we decided to design it to set this requestId at the time of instantiating the request object which means when a user runs the same QueryJobConfiguration twice, they will be sending two requestIds.

Reply:
Thanks for clarifying.  My worry was mostly around cases where someone may retain a reference and use this in unexpected ways.

Reply:
Noted -- thank you!